tiledb/bioimg/converters/base.py

```diff
@@ -276,42 +295,70 @@ def to_tiledb(
             convert_kwargs = dict(
                 reader=reader,
                 rw_group=rw_group,
-                max_tiles=max_tiles,
+                max_tiles=max_tiles.copy(),
```

Comment:
What is the purpose of the `copy` here?

Reply:
The max tiles are modified in case of webp and that causes subsequent level after `level_min` to have larger and larger tiles